src/components/BulkEnrollmentPage/stepper/AddCoursesStep.jsx

```diff
@@ -3,25 +3,28 @@ import PropTypes from 'prop-types';
 import algoliasearch from 'algoliasearch/lite';
 import { InstantSearch, Configure } from 'react-instantsearch-dom';
 import { SearchData, SearchHeader } from '@edx/frontend-enterprise-catalog-search';
+import DismissibleCourseWarning from './DismissibleCourseWarning';
 
 import { configuration } from '../../../config';
 import { ADD_COURSES_TITLE, ADD_COURSE_DESCRIPTION } from './constants';
 
 import CourseSearchResults from '../CourseSearchResults';
 
 const currentEpoch = Math.round((new Date()).getTime() / 1000);
+const maxCourses = 7;
```

Comment:
can we make this all capitalized to make its usage super clear its a constant?

Reply:
+1